citizen_science/citizen_science.md

```diff
@@ -0,0 +1,264 @@
+- donate spare CPU time for computationally intense analyses ([FoldingatHome](https://foldingathome.org/))
+
+
+<div class = "learn-more">
+<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>
+
+Read [an interview with biologist Caren Cooper](https://e360.yale.edu/features/interview_caren_cooper_how_rise_of_citizen_science_is_democratizing_research) about her experience with several citizen science projects, including what she's learned about how participating in citizen science can impact individuals and communities.
+
+</div>
+
+Citizen science is attractive to many researchers not only because of the possibility of outsourcing work on large complex data, but because it provides the potential to democratize science and allow the public more influence over and involvement in the scientific process.
+
+
+<div class = "learn-more">
+<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>
+
+Some projects focus especially on how public involvement in science can put power back in the hands of communities.
+For example, Public Lab works to pursue environmental justice through community science and open technology. From the [Public Lab About page](https://publiclab.org/about):
+
+"When people can easily and reliably track local effects associated with environmental injustices — increased flooding, poor air quality, pollution and destruction of wetlands — they can make better-informed decisions and take action."
+
+</div>
+
+
+The prevalence of citizen science has increased dramatically in recent years, especially as more digital platforms emerge to support it (for example [Zooniverse](https://www.zooniverse.org/), [Globe at Night](https://www.globeatnight.org/), [Foldit](https://fold.it/), and [The Many Brains Project](https://www.testmybrain.org/)).
+
+
+<div class = "history">
+<b style="color: rgb(var(--color-highlight));">Historical context</b><br>
+
+[What Is Citizen Science? – A Scientometric Meta-Analysis](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0147152) explores the history of citizen science projects, including which fields make the most use of citizen science and what scientific output they have produced.
+It includes a meta-analysis of more than 2000 published papers.
+
+</div>
+
+## Why participate in citizen science?
+
+Citizen scientists vary a tremendous amount in terms of the level of dedication they bring to a project, their expertise, and their motivation for participating.
+In some cases, participants are attracted just by the prospect of contributing to a scientific project, especially if it's an issue they care about.
+They also may be motivated by helping their community or themselves with the output of the research.
+
+
+Other times, the project is [designed as a game](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0100662) or includes more traditional incentives like money to attract participants.
```

Comment:
```suggestion
Other times, the project includes more traditional incentives like money to attract participants, or is designed as a game. Using free smartphone games to collect research data can be quite effective, as this [article in PLOS]((https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0100662)  shows.
```

I'm not sure how to make this link both accessible (clear what sort of thing it goes to) and flow nicely, but here is a mediocre suggestion.